src/commcare_cloud/terraform/modules/elasticache-cluster/main.tf

```diff
@@ -28,4 +28,29 @@ resource "aws_elasticache_replication_group" "redis-dev-cluster-0" {
   tags = {
     Name = "${var.namespace}-cache"
   }
+#Redis Log Delivery enginelog configuration
+log_delivery_configuration {
+  destination      = "${var.namespace}-engine-logs"
+  destination_type = "cloudwatch-logs"
+  log_format       = "json"
+  log_type         = "engine-log"
+}
+#Redis Log Delivery slowlog configuration
+log_delivery_configuration {
+  destination      = "${var.namespace}-slow-logs"
+  destination_type = "cloudwatch-logs"
+  log_format       = "json"
+  log_type         = "slow-log"
+}
+
+}
+
+#log group creation for redis engine logs
+resource "aws_cloudwatch_log_group" "elastic-cache-engine-logs" {
+  name = "${var.namespace}-engine-logs"
+}
+
+#log group creation for redis slow logs
+resource "aws_cloudwatch_log_group" "elastic-cache-slow-logs" {
```

Comment:
Feedback from meeting: change `elastic-cache-` to `elasticache-` in these two names

Reply:
sure @dannyroberts I'll look into it and will update the naming convention